### ATAC-seq
1. Shared and distinct biological circuits in effector, memory and exhausted CD8+ T cells revealed by temporal single-cell transcriptomics and epigenetics https://doi.org/10.1038/s41590-022-01338-4
2. Intrinsic bias estimation for improved analysis of bulk and single-cell chromatin accessibility profiles using SELMA https://doi.org/10.1038/s41467-022-33194-z
3. The co-evolution of genome and epigenome in colorectal cancer https://doi.org/10.1038/s41586-022-05202-1
4. Subtype-specific 3D genome alteration in acute myeloid leukaemia https://doi.org/10.1038/s41586-022-05365-x



### scRNA-seq
1. Time-Series Single-Cell RNA-Seq Data Reveal Auxin Fluctuation during Endocycle https://doi.org/10.1093/pcp/pcz228
2. FastRNA: An efficint solution for PCA of single-cell RNA-sequencing data based on a batch-accounting count model https://doi.org/10.1016/j.ajhg.2022.09.008
3. The function and decline of the female reproductive tract at single-cell https://doi.org/10.1101/2022.10.26.513823
4. Anti-CLTA-4 antibodies drive myeloid activation and reprogram the tumor microenviroment through Fc-ganna-R engagement and type I interferon signaling https://doi.org/10.1038/s43018-022-00447-1
5. scAmpi - A versatile pipeline for single-cell RNA-seq analysis from basics to clinics https://doi.org/10.1371/journal.pcbi.1010097
6. Time-resolved single-cell RNA-seq using metabolic RNA labelling https://doi.org/10.1038/s43586-022-00157-z
7. Deep generative modeling for quantifying sample-level heterogeneity in single-cell RNA-sequencing omics
8. IBRAP: Integrated Benchmarking Single-cell RNA-sequencing Analytical Pipeline https://doi.org/10.1101/2022.09.28.
9. Census: accurate, automated, deep, fast, and hierarchical analysis scRNA-seq cell-type annotation https://doi.org/10.1101/2022.10.19.512926
10. devCellPy is a machine learning-enabled pipeline for automated annotation of complex multilayered single-cell transcriptomics data https://doi.org/10.1038/s41467-022-33045-x
11. Deep learning of cross-species single-cell landscapes identifies conserved regulatory programs underlying cell types https://doi.org/10.1038/s41588-022-01197-7



### Trajectory Analysis
1. Tempora: Cell trajectory inference using time-series single-cell RNA sequencing data https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7505465/pdf/pcbi.1008205.pdf
2. scLTNN: Identify the origin and end cells and infer the trajectory of cellular fate automatically `https://doi.org/10.1101/2022.09.28.510020
3. Unraveling causal gene regulation from the RNA velocity graph using Velorama https://doi.org/10.1101/2022.10.18.512766
4. Inferring population dynamics from single-cell RNA sequencing using time series data https://doi.org/10.1038/s41587-019-0088-0



### Gene Expression Analysis
1. A simple gene co-expression analyses workflow powered by tidyverse and graph analyses https://github.com/cxli233/SimpleTidy_GeneCoEx
2. Inference of differential gene regulatory networks from gene expression data using boosted differential trees https://doi.org/10.1101/2022.09.26.509450



### Spatial Transcriptomics
1. Integrated analysis of multimodal single-cell data https://doi.org/10.1016/j.cell.2021.04.048
2. Spatiotemporal transcriptomics reveals pathogenesis of viral myocarditis https://www.nature.com/articles/s44161-022-00138-1
3. The expanding vistas of spatial transcriptomics https://doi.org/10.1038/s41587-022-01448-2
4. Diciphering tissue structure and function using spatial transcriptomics https://doi.org/10.1038/s42003-022-03175-5
5. High-resolution spationtemporal transcriptome mapping of tomato fruit development and ripening https://doi.org/10.1038/s41467-017-02782-9
6. Belayer: Modeling discrete and continuos spatial variation in gene expression from spatially resolved transcriptomics https://doi.org/10.1016/j.cels.2022.09.002
7. Evaluation of cell-cell interaction methods by integrating single-cell interaction RNA sequencing data with spatial information https://doi.org/10.1186/s13059-022-02783-y
8. Modeling intercellular communication in tissues using spatial graphs of cells https://doi.org/10.1038/s41587-022-01467-z
9. Spatially resolved epigenomic profiling of single cells in complex tissues https://doi.org/10.1016/j.cell.2022.09.035
10. Online single-cell data integration through heterogeneous datasets into a common cell-embedding space https://www.nature.com/articles/s41467-022-33758-z
11. Identifying diseas-critical cell types and cellular processes by integrating single-cell RNA-sequencing and human genetics https://doi.org/10.1038/s41588-022-01187-9
12. Space-X: gene co-expression network estimation for spatial transcriptomics https://doi.org/10.1093/bioinformatics/btac645
13. Annotation of spatially resolved single-cell data with STELLAR https://doi.org/10.1038/s41592-022-01651-8
14. Museum of spatial transcriptomics (MUST READ) https://doi.org/10.1038/s41592-022-01409-2 (Study Guide: https://pachterlab.github.io/LP_2021/index.html)
15. Spatial Transcriptomics Coming of Age https://doi.org/10.1038/s41576-019-0129-z
16. Exploring Tissue Architecture using Spatial transcriptomics https://doi.org/10.1038/s41586-021-03634-9 
17. A molecular single-cell lung atlas of lethal COVID-19 https://doi.org/10.1038/s41586-021-03569-1
18. SOLO: Doublet Identification in Single-Cell RNA-Seq via Semi-Supervised Deep Learning https://doi.org/10.1016/j.cels.2020.05.010
19. Spatial components of molecular tissue biology https://doi.org/10.1038/s41587-021-01182-1


### Tools Workflow
1. Spatial transcriptomics workflow using Seurat https://satijalab.org/seurat/index.html
2. Scanpy - Single-Cell Analysis in Python (https://scanpy.readthedocs.io/en/latest/index.html) Tutorial: https://scanpy-tutorials.readthedocs.io/en/latest/index.html
3. Squidpy - Spatial Single Cell Analysis in Python https://squidpy.readthedocs.io/en/stable/index.html
4. Scirpy: a Scanpy extension for analyzing single-cell T-cell receptor-sequencing data https://doi.org/10.1093/bioinformatics/btaa611
5. scvi-tools (single-cell varational inference tools) https://docs.scvi-tools.org/en/stable/index.html#


### Databases
1. PanglaoDB: a web server for exploration of mouse and human single-cell RNA sequencing data https://academic.oup.com/database/article/doi/10.1093/database/baz046/5427041


### Blogs
1. Single-Cell Updates https://scup.netlify.app/
2. Sanbomics https://www.youtube.com/channel/UCuf90yPD_Yx53xZyVLtvRmA
3. Advanced Single-Cell Analysis with Bioconductor https://bioconductor.org/books/3.15/OSCA.advanced/
