### ATAC-seq
1. Shared and distinct biological circuits in effector, memory and exhausted CD8+ T cells revealed by temporal single-cell transcriptomics and epigenetics https://doi.org/10.1038/s41590-022-01338-4
2. Intrinsic bias estimation for improved analysis of bulk and single-cell chromatin accessibility profiles using SELMA https://doi.org/10.1038/s41467-022-33194-z
3. The co-evolution of genome and epigenome in colorectal cancer https://doi.org/10.1038/s41586-022-05202-1
4. Subtype-specific 3D genome alteration in acute myeloid leukaemia https://doi.org/10.1038/s41586-022-05365-x
5. Significant variation in the performance of DNA methylation predictors across data processing and normalization strategies https://doi.org/10.1186/s13059-022-02793-w
6. Multifactorial profiling of epigenetic landscapes at single-cell resolution using MulTI-Tag https://doi.org/10.1038/s41587-022-01522-9
7. Human prefrontal cortex gene regulatory dynamics from gestation to adulthood at single-cell resolution https://doi.org/10.1016/j.cell.2022.09.039
8. 


### scRNA-seq
1. Time-Series Single-Cell RNA-Seq Data Reveal Auxin Fluctuation during Endocycle https://doi.org/10.1093/pcp/pcz228
2. FastRNA: An efficint solution for PCA of single-cell RNA-sequencing data based on a batch-accounting count model https://doi.org/10.1016/j.ajhg.2022.09.008
3. The function and decline of the female reproductive tract at single-cell https://doi.org/10.1101/2022.10.26.513823
4. Anti-CLTA-4 antibodies drive myeloid activation and reprogram the tumor microenviroment through Fc-ganna-R engagement and type I interferon signaling https://doi.org/10.1038/s43018-022-00447-1
5. scAmpi - A versatile pipeline for single-cell RNA-seq analysis from basics to clinics https://doi.org/10.1371/journal.pcbi.1010097
6. Time-resolved single-cell RNA-seq using metabolic RNA labelling https://doi.org/10.1038/s43586-022-00157-z
7. Deep generative modeling for quantifying sample-level heterogeneity in single-cell RNA-sequencing omics
8. IBRAP: Integrated Benchmarking Single-cell RNA-sequencing Analytical Pipeline https://doi.org/10.1101/2022.09.28.
9. Census: accurate, automated, deep, fast, and hierarchical analysis scRNA-seq cell-type annotation https://doi.org/10.1101/2022.10.19.512926
10. devCellPy is a machine learning-enabled pipeline for automated annotation of complex multilayered single-cell transcriptomics data https://doi.org/10.1038/s41467-022-33045-x
11. Deep learning of cross-species single-cell landscapes identifies conserved regulatory programs underlying cell types https://doi.org/10.1038/s41588-022-01197-7
12. A multi-use deep learning method for CITE-seq and single-cell RNA-seq data integration  with cell surface protein prediction and imputation https://doi.org/10.1038/s42256-022-00545-w
13. scCobra: Constrastive cell learning with domain-adaptation for single-cell data integration https://doi.org/10.1101/2022.10.23.513389
14. A Robust Bayesian Approach to Bulk Gene Expression Decolution with Noisy Reference Signatures https://doi.org/10.1101/2022.10.25.513800
15. scGSEA/scMAP: Single-cell gene set enrinchment analysis and transfer learning for functional annotation of scRNA-seq data https://doi.org/10.1101/2022.10.24.513476
16. Transcriptomics Profiling of Plasma Extracellular Vesicles Enables Reliable Annotation Cancer-specific Transcriptome and Molecular Expression Subtype https://doi.org/10.1101/2022.10.27.514047
17. Deep learning explains the biology of branched glycans from single-cell sequencing data https://doi.org/10.1016/j.isci.2022.105163
18. Three-dimensional imaging and single-cell transcriptomics of the human kidnet implicate pertubation of lymphatics in alloimmunity https://doi.org/10.1101/2022.10.28.514222 https://twitter.com/daniyal_jafree/status/1587411157115174918?s=20&t=0MfLopjcBYUIIcziFPu2GQ
19. Pan-cancer single-cell landscape of tumor-infiltrating T cells https://doi.org/10.1126/science.abe6474
20. Pan-cancer single-cell analysis reveals the heterogeneity and plasticity of cancer-associated fibroblasts in the tumor microenviroment https://doi.org/10.1038/s41467-022-34395-2 https://twitter.com/simocristea/status/1589669703731253249?s=20&t=0MfLopjcBYUIIcziFPu2GQ
21. An integrated Bayesian framework for multi-omics prediction and classification
22. Integrated regulatory network analysis of single-cell transcriptomes and chromatin accessibility profiles https://doi.org/10.1016/j.isci.2022.105359
23. Transmorph: A unifying computational framework for single-cell data integration https://doi.org/10.1101/2022.11.02.514912
24. Single-cell RNA-sequencing of peripheral blood mononuclear cells reveals widespread, context-specific gene expression regulation upon exposure https://doi.org/10.1038/s41467-022-30893-5
25. Deep transfer learning of cancer drug responses by integrating bulk and single-cell RNA-sequencing RNA-seq data https://doi.org/10.1038/s41467-022-34277-7
26. Three-dimensional imaging and single-cell transcriptomics of the human kidney implicate pertubation of lymphatcs in alloimmunity https://doi.org/10.1101/2022.10.28.514222
27. Pre-training Enhanced Spatial-Temporal Graph Neural Network for Multivariate Time Series Forecasting https://arxiv.org/pdf/2206.09113.pdf
28. The gene expression landscape of the human locus coeruleus revealed by single-nucleus and spatially-resolved transcriptomics https://doi.org/10.1101/2022.10.28.514241
29. scAN10: A reproducible and standardized pipeline for processing 10X single cell RNAseq data https://doi.org/10.1101/2022.11.07.515546
30. Direct Proteomic Quantification of the Secretome of Activated Immune Cells https://doi.org/10.1126/science.1232578
31. Heterogeneous plasma cells and long-lived subsets in response to immunization, autoantigen and microbiota https://doi.org/10.1038/s41590-022-01345-5


### Trajectory Analysis
1. Tempora: Cell trajectory inference using time-series single-cell RNA sequencing data https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7505465/pdf/pcbi.1008205.pdf
2. scLTNN: Identify the origin and end cells and infer the trajectory of cellular fate automatically `https://doi.org/10.1101/2022.09.28.510020
3. Unraveling causal gene regulation from the RNA velocity graph using Velorama https://doi.org/10.1101/2022.10.18.512766
4. Inferring population dynamics from single-cell RNA sequencing using time series data https://doi.org/10.1038/s41587-019-0088-0
5. Unravelling causal gene regulation from the RNA velocity graph Velorama https://doi.org/10.1101/2022.10.18.512766


### Gene Expression Analysis
1. A simple gene co-expression analyses workflow powered by tidyverse and graph analyses https://github.com/cxli233/SimpleTidy_GeneCoEx
2. Inference of differential gene regulatory networks from gene expression data using boosted differential trees https://doi.org/10.1101/2022.09.26.509450
3. Deep model predictive control of gene expression in thousands of single cells https://doi.org/10.1101/2022.10.28.514305
4. Gene Expression clustering using Gaussian mixture modelling https://doi.org/10.1186/s12859-022-05006-0
5. MTM: a multi-task framework to predict individualized tissue gene expression profiles https://doi.org/10.1101/2022.10.19.512838
6. spliceJAC: transition genes and state-specific gene regulation from single-cell transcriptome data https://doi.org/10.15252/msb.202211176


### Spatial Transcriptomics
1. Integrated analysis of multimodal single-cell data https://doi.org/10.1016/j.cell.2021.04.048
2. Spatiotemporal transcriptomics reveals pathogenesis of viral myocarditis https://www.nature.com/articles/s44161-022-00138-1
3. The expanding vistas of spatial transcriptomics https://doi.org/10.1038/s41587-022-01448-2
4. Diciphering tissue structure and function using spatial transcriptomics https://doi.org/10.1038/s42003-022-03175-5
5. High-resolution spationtemporal transcriptome mapping of tomato fruit development and ripening https://doi.org/10.1038/s41467-017-02782-9
6. Belayer: Modeling discrete and continuos spatial variation in gene expression from spatially resolved transcriptomics https://doi.org/10.1016/j.cels.2022.09.002
7. Evaluation of cell-cell interaction methods by integrating single-cell interaction RNA sequencing data with spatial information https://doi.org/10.1186/s13059-022-02783-y
8. Modeling intercellular communication in tissues using spatial graphs of cells https://doi.org/10.1038/s41587-022-01467-z
9. Spatially resolved epigenomic profiling of single cells in complex tissues https://doi.org/10.1016/j.cell.2022.09.035
10. Online single-cell data integration through heterogeneous datasets into a common cell-embedding space https://www.nature.com/articles/s41467-022-33758-z
11. Identifying diseas-critical cell types and cellular processes by integrating single-cell RNA-sequencing and human genetics https://doi.org/10.1038/s41588-022-01187-9
12. Space-X: gene co-expression network estimation for spatial transcriptomics https://doi.org/10.1093/bioinformatics/btac645
13. Annotation of spatially resolved single-cell data with STELLAR https://doi.org/10.1038/s41592-022-01651-8
14. Museum of spatial transcriptomics (MUST READ) https://doi.org/10.1038/s41592-022-01409-2 (Study Guide: https://pachterlab.github.io/LP_2021/index.html)
15. Spatial Transcriptomics Coming of Age https://doi.org/10.1038/s41576-019-0129-z
16. Exploring Tissue Architecture using Spatial transcriptomics https://doi.org/10.1038/s41586-021-03634-9 
17. A molecular single-cell lung atlas of lethal COVID-19 https://doi.org/10.1038/s41586-021-03569-1
18. SOLO: Doublet Identification in Single-Cell RNA-Seq via Semi-Supervised Deep Learning https://doi.org/10.1016/j.cels.2020.05.010
19. Spatial components of molecular tissue biology https://doi.org/10.1038/s41587-021-01182-1
20. Unsupervised discovery of tissue architecture in multiplexed imaging https://doi.org/10.1038/s41592-022-01657-2
21. De novo analysis of bulk RNA-seq data at spatially resolved single-cell resolution https://doi.org/10.1038/s41467-022-34271-z
22. Modelling intercellular communication in tissues using spatial graphs of cells https://doi.org/10.1038/s41587-022-01467-z
23. 


### Tools Workflow
1. Spatial transcriptomics workflow using Seurat https://satijalab.org/seurat/index.html
2. Scanpy - Single-Cell Analysis in Python (https://scanpy.readthedocs.io/en/latest/index.html) Tutorial: https://scanpy-tutorials.readthedocs.io/en/latest/index.html
3. Squidpy - Spatial Single Cell Analysis in Python https://squidpy.readthedocs.io/en/stable/index.html
4. Scirpy: a Scanpy extension for analyzing single-cell T-cell receptor-sequencing data https://doi.org/10.1093/bioinformatics/btaa611
5. scvi-tools (single-cell varational inference tools) https://docs.scvi-tools.org/en/stable/index.html#
6. Semisupervised adversarial neural networks for single-cell classification (scnym: https://doi.org/10.1101/gr.268581.120)

### Databases
1. PanglaoDB: a web server for exploration of mouse and human single-cell RNA sequencing data https://academic.oup.com/database/article/doi/10.1093/database/baz046/5427041

### Blogs
1. Single-Cell Updates https://scup.netlify.app/
2. Sanbomics https://www.youtube.com/channel/UCuf90yPD_Yx53xZyVLtvRmA
3. Advanced Single-Cell Analysis with Bioconductor https://bioconductor.org/books/3.15/OSCA.advanced/
